ts/features/wallet/satispay/SatispayWalletPreview.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from "react";
+import { connect } from "react-redux";
+import { Dispatch } from "redux";
+import { nullType } from "io-ts";
+import satispayImage from "../../../../img/wallet/cards-icons/satispay.png";
+import { Body } from "../../../components/core/typography/Body";
+import { IOStyles } from "../../../components/core/variables/IOStyles";
+import I18n from "../../../i18n";
+import { GlobalState } from "../../../store/reducers/types";
+import { CardPreview } from "../component/CardPreview";
+
+type Props = ReturnType<typeof mapDispatchToProps> &
+  ReturnType<typeof mapStateToProps>;
+
+/**
+ * A card preview for a bancomat card
+ * @param props
+ * @constructor
+ */
+const SatispayWalletPreview: React.FunctionComponent<Props> = props => (
+  <CardPreview
+    left={
+      <Body style={IOStyles.flex} numberOfLines={1}>
+        {I18n.t("wallet.methods.satispay.name")}
+      </Body>
+    }
+    image={satispayImage}
+    onPress={() => props.navigateToSatispayDetails()}
```

Comment:
```suggestion
    onPress={props.navigateToSatispayDetails}
```

Reply:
@Undermaken I would accept the suggestion, but this is going to be replaced by the following implementation of correct navigation action see [here](https://github.com/pagopa/io-app/pull/2458/files#diff-d4a25fe5860c665ade63a83a9b2717c0a07c2730801becbbb220b635f472b865R34) 😄

Reply:
No problem ;)
Leave it as it is